private/pkg/manifest/module_test.go

```diff
@@ -43,3 +45,60 @@ func TestDigestFromBlobHash(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, digestFromContent.String(), digestFromBlobHash.String())
 }
+
+func TestNewMemoryBlob(t *testing.T) {
+	t.Parallel()
+	const content = "some file content"
```

Comment:
For less casting?

```suggestion
	const content = []byte("some file content")
```

Reply:
I wish, but slices cannot be consts :/

Reply:
Ah, and you're preferring const over the many cast. Works for me.